src/main/scala/github/gphat/censorinus/dogstatsd/Encoder.scala

```diff
@@ -62,14 +62,15 @@ object Encoder extends MetricEncoder {
 
   def encodeEvent(sc: EventMetric): String = {
     val sb = new StringBuilder()
+    val escapedText = sc.text.replace("\n", "\\\\n")
```

Comment:
Per [the Datadog escaping method](https://github.com/DataDog/dd-agent/blob/master/aggregator.py#L506), do we want to also escape `m:` to `m\:`? (Also, ha, seeing that escaping would have saved some time when we ran into that one :laughing: )